ipaserver/install/dogtaginstance.py

```diff
@@ -320,9 +320,9 @@ def secure_ajp_connector(self):
         if len(connectors) == 0:
             return
 
-        # AJP protocol is at version 1.3. Assume there is only one as
-        # Dogtag only provisions one.
-        connector = connectors[0]
+        # Wether or not we should rewrite the tomcat server.xml file with
```

Comment:
nit: Wether -> Whether

Reply:
```suggestion
        # Whether or not we should rewrite the tomcat server.xml file with
```

Reply:
@rcritten Feel free to commit suggestion if you want to fix this prior to push.